api/candlepin-api-spec.yaml

```diff
@@ -555,6 +555,57 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /cloud:
+    post:
```

Comment:
A note here: this was not accurately ported. The original code had `@Path("/cloud")` on the class and `@Path("authorize")` on the method, which would effectively be a `POST /cloud/authorize`, while the way you ported would result in a `POST /cloud`